admin/App.js

```diff
@@ -36,49 +37,44 @@ export class App extends Component {
     // Implementation
     //------------------
     renderNavBar() {
-        return navbar({
-            items: [
-                navbarGroup({
-                    align: 'left',
-                    items: [
-                        Icon.gears({size: '2x'}),
-                        navbarHeading(`${XH.appName} Admin`)
-                    ]
+        return appBar({
+            icon: Icon.gears({size: '2x'}),
+            title: `${XH.appName} Admin`,
+            leftItems: [
+                tabSwitcher({
+                    model: this.model.tabs,
+                    large: false
+                })
```

Comment:
The one feature question that's been in my mind re. this comp is if it should also bundle up the creation of a tabSwitcher - could support a `tabContainerModel` prop that, if given, creates this automatically.  That might be a useful convenience and help to further standardize.  (Also looking at the need for `large: false` here which is a bit unfortunate if we do want to use this pattern frequently.)

Reply:
Yeah was thinking the same, makes sense to standardize on the main app nav switcher I think.

Re: `large: false` on the `tabSwitcher`, I'd prefer that we not default to large tabs in `tabSwitcher`. I kept it in there when I did the TabContainer re-work. Seems like it was mostly there because the top-level tabs were typically used for the app nav, with potentially 1 level of nesting using vertical tabs, so the larger horizontal tabs made sense. Any objection to just taking large tabs for horizontal out of `tabSwitcher`?